clients/spark/core/src/main/hadoop3/scala/io/treeverse/clients/conditional/S3ClientBuilder.scala

```diff
@@ -1,11 +1,19 @@
 package io.treeverse.clients.conditional
 
-import com.amazonaws.ClientConfiguration
+import com.amazonaws.retry.PredefinedRetryPolicies.SDKDefaultRetryCondition
+import com.amazonaws.retry.RetryPolicy
 import com.amazonaws.services.s3.{AmazonS3, AmazonS3ClientBuilder}
+import com.amazonaws.{AmazonClientException, AmazonWebServiceRequest, ClientConfiguration, SdkClientException}
 import io.treeverse.clients.StorageUtils.S3.createAndValidateS3Client
 import org.apache.hadoop.conf.Configuration
 import org.slf4j.{Logger, LoggerFactory}
 
+class S3RetryCondition extends SDKDefaultRetryCondition {
```

Comment:
Rather than repeat this, can we build it in `io.treeverse.clients` and import from there?